client/src/client.rs

```diff
@@ -1061,21 +1045,21 @@ impl Client {
                 ));
             }
         }
-        Ok(None)
+        Ok(vec![])
```

Comment:
```suggestion
Vec::new()
```

let's not use macros where not required